services/frontend-service/src/ui/components/CommitInfo/CommitInfo.tsx

```diff
@@ -15,15 +15,30 @@ along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>
 Copyright 2023 freiheit.com*/
 
 import { TopAppBar } from '../TopAppBar/TopAppBar';
-import React from 'react';
+import React, { useCallback } from 'react';
 import { GetCommitInfoResponse, Event, LockPreventedDeploymentEvent_LockType } from '../../../api/api';
+import { Button } from '../button';
+import './CommitInfo.scss';
 
 type CommitInfoProps = {
     commitInfo: GetCommitInfoResponse | undefined;
 };
 
 export const CommitInfo: React.FC<CommitInfoProps> = (props) => {
     const commitInfo = props.commitInfo;
+
+    const onClickPrevious = useCallback((): void => {
+        if (commitInfo !== undefined) {
+            window.location.href = commitInfo?.previousCommitHash;
+        }
+    }, [commitInfo]);
+
+    const onClickNext = useCallback((): void => {
+        if (commitInfo !== undefined) {
+            window.location.href = commitInfo?.nextCommitHash;
+        }
+    }, [commitInfo]);
```

Comment:
Please use normal links (`<a href...`) instead of callbacks.
Links are there to reach places via a url. Buttons/callbacks are more complicated and not needed in this case.